config/final-mkdocs.yml

```diff
@@ -0,0 +1 @@
+INHERIT: 1-redirects.yml
```

Comment:
This file will need to be updated to be `2-<description>.yml` if any additional chained inherits are required in future